docs/workflow_tasks/move_granules.md

```diff
@@ -0,0 +1,57 @@
+---
+id: move_granules
+title: Move Granules
+hide_title: true
+---
+
+# Move Granules
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+This task utilizes the incoming config.input_granules and the task input list of s3 URIs along with the rest of the configuration objects to do the following for a list of files assigned to a single collection:
+
+* Take the list of incoming files and sort them into a list of granule objects.  Assign files to existing granules where appropriate.
+
+* Move granules from their 'staging' location to the final location (as configured )
+
+* Update the config.input_granules object with the new file locations.
+
+* If the granule has a ECHO10/UMM CMR file(.cmr.xml or .cmr.json) file included in the config.input_granules, open that file and update access locations, and add it to the appropriate access URL category for the CMR filetype as defined by granule CNM filetype.   Set the CMR file to 'metadata' in the output granules object and add it to  the granule files if it's not already present.
```

Comment:
I would make this a bulleted list underneath this bullet.

Reply:
Good idea!   2da45d0c8